packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -96,18 +109,36 @@ class EmbeddedContentConsumer extends React.Component {
       onAuthorize,
       options,
       eventHandlers,
+      intl,
       ...customProps
     } = this.props;
 
     return (
-      <div
-        {...customProps}
-        ref={(element) => { this.embeddedContentWrapper = element; }}
-      />
+      <div className={cx('container')}>
+        <VisuallyHiddenText
+          className={cx('visually-hidden-text')}
+          text={intl.formatMessage(
+            { id: 'Terra.embeddedContentConsumer.beginEmbeddedContent' },
+            { name: options?.iframeAttrs?.title },
```

Comment:
Could you rebase with main?

Reply:
@kolkheang - I am assuming we need to set the title directly as the name here instead of `options?.iframeAttrs?.title`? Is my understanding right?

Reply:
@kolkheang - I have updated the name to be from title prop directly, please let me know if I need to revert this - https://github.com/cerner/terra-framework/pull/1804/commits/3c5bc88cbdd595f9ee67b882452e6aee92068f86#diff-77b1c1b39ad95249bd3956ef509a0995b4687f8c94ba87a3110f3e6b1ce8ccfeR155